x-pack/plugin/async-search/src/internalClusterTest/java/org/elasticsearch/xpack/search/AsyncSearchErrorTraceIT.java

```diff
@@ -32,7 +33,7 @@ protected boolean addMockHttpTransport() {
 
     @Override
     protected Collection<Class<? extends Plugin>> nodePlugins() {
-        return List.of(AsyncSearch.class);
+        return List.of(AsyncSearch.class, MockTransportService.TestPlugin.class);
```

Comment:
I'd prefer we added these things to `super.nodePlugins()` with `appendToCopy` or similar. Doesn't really matter in this case right now but adding stuff to `ESIntegTestCase#nodePlugins` everywhere for an experiment is sometimes handy.

Reply:
Makes sense :) done :)